.codex/skills/openspec-verify-change/SKILL.md

```
@@ -1,13 +1,17 @@
-description: Verify implementation matches change artifacts before archiving
+name: openspec-verify-change
+description: Verify implementation matches change artifacts. Use when the user wants to validate that implementation is complete, correct, and coherent before archiving.
+license: MIT
+compatibility: Requires openspec CLI.
+metadata:
+  author: openspec
+  version: "1.0"
+  generatedBy: "1.3.0"
 ---
 
 Verify that an implementation matches the change artifacts (specs, tasks, design).
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)